class AlreadyExistsException(BaseException):
    pass

class NotFoundException(BaseException):
    pass
